packages/appstore/src/components/modals/cashier-wallet-modal/cashier-wallet-modal.scss

```diff
@@ -0,0 +1,177 @@
+.dc-modal__container_cashier-wallet-modal {
+    position: fixed;
+    inset: 0;
+    min-height: calc(100vh - 84px) !important;
+    min-width: 100vw !important;
+    margin: 48px 0 36px;
+    border-radius: unset;
+    background-color: var(--general-main-1);
+
+    @include mobile {
+        min-height: 100vh !important;
+        min-width: 100vw !important;
+        margin: 0;
+        z-index: 100;
+    }
+
+    .cashier-wallet-modal {
+        &__header {
+            display: flex;
+            position: relative;
+            height: 16rem;
+            padding: 2.4rem 4rem 7.2rem;
+
+            @include mobile {
+                height: 12.2rem;
+                padding: 1.6rem 1.6rem 5.6rem;
+            }
+
+            &--hide-title {
+                height: 9.8rem;
+                align-items: center;
+                justify-content: space-between;
+            }
+
+            &__demo--light:before {
+                background-color: $color-white;
+            }
+
+            &__demo--dark:before {
+                background-color: $wallet-demo-dark-watermark-color;
+            }
+
+            &__demo:before {
+                content: '';
+                display: block;
+                position: absolute;
+                left: 0;
+                top: 0;
+                width: 100%;
+                height: 100%;
+                opacity: 0.2;
+                background-repeat: repeat;
+                background-position: 3% 30%;
+                border-radius: $BORDER_RADIUS * 4;
+                -webkit-mask-image: url('../../../public/images/wallet-demo-bg.svg');
+                mask-image: url('../../../public/images/wallet-demo-bg.svg');
+            }
+
+            &__title-wrapper {
+                position: relative;
+            }
+
+            &__title {
+                display: flex;
+                align-items: center;
+
+                &-wallet {
+                    padding-right: 0.8rem;
+                    color: var(--text-general);
+
+                    &-demo-dark {
```

Comment:
better to not use classed for this `light/dark` variation and use var from `theme.scss`. if there is no variation for this color you can add new one